### DISCUSSION TITLE
Pager for List, Tags, Search

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#12215] [Q&A] [answered]
Hello,

I am having some issues with the default pager on Taxonomies, where it will not go past the third page

you can click on the next button and go to the second page of the results, but it will not go past that. When I hit the next button again, it takes me back to the taxonomy list.

Ex. 
Category List -> Click on category -> List of contents for that category show -> Go to Secon page -> try to go to third page -> takes me back to category list.

The question is if the controller can be overridden with my own custom one. or if the pager type can be changed to full

**Comment by Skrypt** [comment, marked as answer]
The issue is fixed in the main branch as per: https://github.com/OrchardCMS/OrchardCore/pull/11726
This fix is included in 1.4.0.
